packages/json-schemas/schemas/call_data_schema.json

```diff
@@ -19,6 +19,24 @@
         "nonce": {
             "type": "number",
             "minimum": 0
+        },
+        "overrides": {
+            "type": "object",
+            "additionalProperties": {
+                "type": "object",
+                "properties": {
+                    "code": {
+                        "type": "string",
+                        "pattern": "^0x[0-9a-f]*$"
+                    },
+                    "nonce": {
+                        "oneOf": [{ "$ref": "/numberSchema" }, { "$ref": "/jsNumberSchema" }]
+                    },
+                    "value": {
```

Comment:
is this `balance` and a `wholeNumberSchema` (wei)?

Reply:
d-doh